test/functional/sendheaders.py

```diff
@@ -92,9 +92,9 @@
 
 direct_fetch_response_time = 0.05
 
-class BaseNode(SingleNodeConnCB):
+class BaseNode(NodeConnCB):
```

Comment:
Should also fix `dip4-coinbasemerkleroots.py` and `llmq-is-cl-conflicts.py` the same way as it's done here

Reply:
https://github.com/dashpay/dash/pull/2771/commits/50a99de1f09f9bcfa242c551028ce7759a689639